docs/photoniq/faas/faas-structure.md

```diff
@@ -0,0 +1,126 @@
+---
+sidebar_position: 9
+title: FaaS Functions Structure
+---
+
+FaaS functions can be written in `Rust` or `JavaScript`. This documentation explains the code templates and settings for these functions.
+
+## Rust Function
+
+Each function receives an `_req: HttpRequest` struct and returns a `Result<HttpResponse>` struct. These structs are available in the `PhotonIQ FaaS SDK`. The `HttpResponseBuilder` crate allows you to build various types of responses.
+
+This is the basic boilerplate code required for each function. Users can add their custom code in the section marked with `<USER CODE IS WRITTEN HERE>`.
+```rust
+use anyhow::Result;
+use photoniq_faas_sdk::{
+    binding::{
+        http::{HttpRequest, HttpResponse, HttpResponseBuilder},
+        log,
+    },
+    function_handler,
+};
+use std::env;
+
+#[function_handler]
+fn handler(_req: HttpRequest) -> Result<HttpResponse> {
+    < USER CODE IS WRITTEN HERE >
+}
+```
+
+Below is an example of a function template code:
+
+```rust
+use anyhow::Result;
+use photoniq_faas_sdk::{
+    binding::{
+        http::{HttpRequest, HttpResponse, HttpResponseBuilder},
+        log,
+    },
+    function_handler,
+};
+use std::env;
+
+#[function_handler]
+fn handler(_req: HttpRequest) -> Result<HttpResponse> {
+    let message = env::var("MESSAGE").unwrap_or_else(|_| String::from("Missing message"));
+
+    let response = format!(
+        "PhotonIQ FaaS function is working!
+
+Env Var MESSAGE = {}",
+        message
+    )
+    .to_string();
+
+    // Log messages with the following methods: error!, warn!, info!, debug!, trace!
+    log::error!("[{}:{}] Log an [ERROR] message", file!(), line!());
+
+    Ok(HttpResponseBuilder::ok().body(response.into()).build())
+}
+```
+
+## JavaScript Function
+Functions based on JavaScript work out of the box PhotonIQ function. The service integrates a JavaScript interpreter compiled into a WebAssembly module. Currently, the interpreter we support is `quickjs` and we are working on adding new ones.
```

Comment:
This whole section is confusing:
1. Functions based on JavaScript work out of the box PhotonIQ function. (JS function work P function)?
Not sure what we are trying to say here.
2. Is all this stuff about quickjs needed?  Why are we explaining in the documentation how we implemented something?

Reply:
@joubert-macrometa, you are right. This needs to go. Thanks!